packages/grid/data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -127,13 +120,13 @@ describe('<DataGrid /> - Keyboard', () => {
   });
 
   const KeyboardTest = (props: { nbRows?: number }) => {
-    const data = useData(props.nbRows || 100, 20);
+    const data = useData(props.nbRows || 20, 20);
     const transformColSizes = (columns: GridColumns) =>
       columns.map((column) => ({ ...column, width: 60 }));
 
     return (
       <div style={{ width: 300, height: 360 }}>
-        <DataGrid rows={data.rows} columns={transformColSizes(data.columns)} />
+        <DataGrid autoHeight rows={data.rows} columns={transformColSizes(data.columns)} />
```

Comment:
why autoheight?

x times

Reply:
You can start with this thread https://github.com/mui-org/material-ui-x/issues/1151#issuecomment-812068616. If it's still unclear, happy to answer follow-up questions.

Reply:
Read the thread and I wouldn't use it in all the tests for the reason I mentioned in my previous comment.
AutoHeight is not the default approach and probably not the most popular. 
Also one of the most critical bit lies in the virtualization.

Reply:
What do you think about testing with autoHeight only in jsdom?

Reply:
Ok, I will move with this option, seems like a great compromise